src/parser/datetime.rs

```diff
@@ -68,6 +70,92 @@ pub(crate) fn tokenize_interval(value: &str) -> Result<Vec<IntervalToken>, Parse
     Ok(toks)
 }
 
+fn tokenize_timezone(value: &str) -> Result<Vec<IntervalToken>, ParserError> {
+    let mut toks: Vec<IntervalToken> = vec![];
+    let mut num_buf = String::with_capacity(4);
+    /* If the timezone string has a colon, we need to parse all numbers naively, otherwise we need to parse long sequences of digits as [..hhhhmm] */
```

Comment:
nit: rust idiomatically uses line comments in all cases, and wraps them to 80 chars in most.

Reply:
changed to line args
just want to confirm - cargo fmt seems like it won't do the line wrapping do you want me to go ahead and do that manually?

Reply:
fixed manually